sycl/test-e2e/Matrix/joint_matrix_bf16_fill_k_cache_impl.hpp

```diff
@@ -18,19 +18,39 @@ constexpr unsigned int testIterations = 100;
 // start recording time after X iterations
 constexpr unsigned int recordThresh = 10;
 
+#ifndef MATRIX_SIZE
 #define MATRIX_SIZE 256
+#endif
 
+#ifndef tM
 #define tM 8
+#endif
+#ifndef tN
+#define tN TN
```

Comment:
where is TN definition?

Reply:
in .cpp file

Reply:
We moved TN to .cpp file because it changes between DG2 and PVC. so we can still use same .hpp file for XMX8 folder as well